themes/default/content/blog/oidc-with-azure/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Oidc with Azure"
+
+# The date represents the post's publish date,
+# and by default corresponds with the date this file was generated.
+# Posts with future dates are visible in development,
+# but excluded from production builds.
+# Use the time and timezone-offset portions of of this value
+# to schedule posts for publishing later.
+date: 2023-05-17T12:36:04-07:00
+
+# Use the meta_desc property to provide a brief summary
+# (one or two sentences) of the content of the post,
+# which is useful for targeting search results or social-media previews.
+# This field is required or the build will fail the linter test.
+# Max length is 160 characters.
+meta_desc: A guide to using OIDC authentication with Azure in Pulumi
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect ratio
+# has been provided for you.
+meta_image: meta.png
+
+# At least one author is required.
+# The values in this list correspond with the `id` properties
+# of the team member files at /data/team/team.
+# Create a file for yourself if you don't already have one.
+authors:
+    - thomas-kappler
+
+# At least one tag is required.
+# Lowercase, hyphen-delimited is recommended.
+tags:
+    - azure
+
+---
+
+Recently, we’ve released a long requested feature: OIDC authentication for Azure. It's available in both the native and the classic provider. Let’s dig in to what it is, how it works, and why it’s useful.
+
+<!--more-->
+
+{{% notes type="info" %}}
+[Pulumi announced OIDC support for Deployments](https://www.pulumi.com/blog/oidc-blog/) in January. This article is about the same OIDC protocol, this time used to authenticate your Pulumi program to access Azure resources.
+{{% /notes %}}
+
+### The What and Why of OIDC
+
+OIDC stands for OpenID Connect, which is a standardized protocol for federated identity. This means that if you maintain an identity with two service providers, you can tell one of them to trust the other. Then, you can use credentials for the trusted service to get credentials for the other.
+
+As an example, we’ll use a Pulumi program running in CI on GitHub Workflows, creating Azure resources in the process. Without OIDC, this program would need a secret to authenticate with Azure - a client secret or a certificate. With OIDC, your program doesn't need a secret. It can send its GitHub token to Azure instead.  Based on the trust relationship between the two, Azure will exchange the GitHub token for an Azure token.
```

Comment:
```suggestion
For example, we’ll use a Pulumi program running in CI on GitHub Workflows, creating Azure resources. Without OIDC, this program would need a secret to authenticate with Azure - a client secret or a certificate. With OIDC, your program doesn't need a secret. It can send its GitHub token to Azure instead. Based on the trust relationship between the two, Azure will exchange the GitHub token for an Azure token.
```